service-commands/scripts/bake-dev-image.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
```

Comment:
`/usr/bin/env bash`? just in case we run something special in Circle CI at one point?